kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/task/InventoryIncrementalTasksRunner.java

```diff
@@ -49,6 +49,7 @@ public class InventoryIncrementalTasksRunner implements PipelineTasksRunner {
     
     private final Collection<IncrementalTask> incrementalTasks;
     
+    @Getter
     private final PipelineJobAPI jobAPI;
```

Comment:
Is `@Getter` required?

Reply:
Maybe subclass only have inventory task, then it's will use jobAPI methods, just like `stop()`